Description: Remove java-8 JREs from list of possible Java commands.
 josm is no longer built with openjdk-8-sdk and requires java-9-jre or newer at runtime.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/linux/tested/usr/bin/josm
+++ b/linux/tested/usr/bin/josm
@@ -13,7 +13,6 @@ ARCH="`dpkg --print-architecture 2> /dev
 if dpkg --get-selections 'openjdk-*-jre' | grep install$ > /dev/null ; then
 	JAVA_CMDS="$JAVA_HOME/bin/java"
 	JAVA_CMDS="${JAVA_CMDS} /usr/lib/jvm/java-11-openjdk/bin/java /usr/lib/jvm/java-11-openjdk-$ARCH/bin/java"
-	JAVA_CMDS="${JAVA_CMDS} /usr/lib/jvm/java-8-openjdk/bin/java /usr/lib/jvm/java-8-openjdk-$ARCH/bin/java"
 	JAVA_CMDS="${JAVA_CMDS} /usr/lib/jvm/java-10-openjdk/bin/java /usr/lib/jvm/java-10-openjdk-$ARCH/bin/java"
 	JAVA_CMDS="${JAVA_CMDS} /usr/lib/jvm/java-9-openjdk/bin/java /usr/lib/jvm/java-9-openjdk-$ARCH/bin/java"
 else
